cards.js

```diff
@@ -1,5 +1,5 @@
-DECK_DEFINITONS = 

```

Comment:
I just forced autocrlf=true on my home Windows machines, and I think it automatically does that :p